App/Containers/Dashboard.js

```diff
@@ -199,7 +217,8 @@ export default class Dashboard extends React.Component {
       )
     }
 
-    if (this.state[StorageKeys.FITBIT_ACCESS_TOKEN] !== null && this.state[StorageKeys.CAT_ID] !== null) {
+    if ((this.state[StorageKeys.FITBIT_ACCESS_TOKEN] !== null && this.state[StorageKeys.FITBIT_ACCESS_TOKEN] !== '') &&
+    (this.state[StorageKeys.CAT_ID] !== null && this.state[StorageKeys.FITBIT_ACCESS_TOKEN] !== '')) {
```

Comment:
Nitpick: if we're going to use these conditions a lot maybe make a function? I know they're used below as well.